TERMS.md

```diff
@@ -20,12 +20,18 @@ A dictionary of Free and Open Source software terms
    Software](#Free_Software) [Licenses](#License), e.g. the
    [GPL](#GPL) family. See also [Copyright](#Copyright)
 
-***<a name="Copyright">Copyright</a>*** is a legal construct to grant
+***<a name="Copyright">Copyright</a>*** is a legal construct to grant the creator of a work exclusive rights to the work.
+
+***<a name="Cumulative_license">Cumulative license</a>*** refers to
+when a software is released under more than one license, e.g. `Apache-2.0 AND LGPL-2.1-only`, and all the
+licenses must be adhered to. This situation often is the result of combining many software comonents rather than chosing to license a software under a cumulative license. See also [Dual license](#Dual_license).
```

Comment:
I would probably say "This situation is often" rather than "often is"
small typo, comonents => components

However, I am not sure I completely understand what this paragraph is trying to say.

Cumulative license: when sw is released under multiple licensen and all terms must be respected. This situation (the cumulativeness I gather) is the result of combining many components, rather than chosing to license a software under a ... cumulative license?! (Is this not the same situation we started out describing in the beginning of the paragraph?)